src/views/Home/components/Banners/IFOBanner.tsx

```diff
@@ -86,15 +94,43 @@ const RightWrapper = styled.div`
   }
 `
 
+const activeIfo = ifosConfig.find((ifo) => ifo.isActive)
+
 const IFOBanner = () => {
   const { t } = useTranslation()
+  const currentBlock = useCurrentBlock()
+
+  const { data: currentIfoBlocks = { startBlock: 0, endBlock: 0 } } = useSWRImmutable(
```

Comment:
Could we make this hook reusable ? I'm going to need it for updating the status in the IFO menu item

Reply:
Done